modular_skyrat/modules/sec_haul/code/guns/bullets.dm

```diff
@@ -316,46 +315,46 @@
 //FMJ | JHP | IHDF | RUBBER//
 ///////////////////////////////
 /obj/item/ammo_casing/caseless/b473
```

Comment:
```suggestion
/obj/item/ammo_casing/caseless/b47
```

Reply:
Also not really necessary

Reply:
It's to prevent shit being impossible to find, because when staff is trying to spawn in something, they don't have access to see the name of it, only the typepath, which makes it a pain to find some stuff, sometimes. This would likely be such a case.